halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/security/authn/EditOAuth2Command.java

```diff
@@ -74,6 +82,10 @@ protected AuthnMethod editAuthnMethod(OAuth2 authnMethod) {
 
     authnMethod.setProvider(provider != null ? provider : authnMethod.getProvider());
 
+    if (!userInfoRequirements.isEmpty()) {
```

Comment:
`UserInfoRequirements` extends `HashMap<String, String>`, which is great for two reasons:

1. It's required to use `DynamicParameter` 
1. Both key and value must be define by the user in the config.